### PR TITLE
CIF-808 - fix sticky Venia header

### PR DESCRIPTION
**Pull request by mhaack**
## Description

Fix the Venia header and make it sticky.

## Related Issue

CIF-808

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [x] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css [hunk 1]
```diff
@@ -44,6 +44,13 @@
...
 }
 
+.header {
+    position: -webkit-sticky;
+    position: sticky;
+    top: 0;
+    z-index: 10;
+}
+
 .header__root {
```
Comment (cjelger) [inline ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css]: I'm not a css expert, but adding replacing `position: fixed` and adding `width: 100%` to `header__root` also fixes that problem

Comment (cjelger) [reply ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css]: It seems "sticky" is useful when you want something to scroll for a while an then at some point stick ... but hey I'm really not a css guru!

**Comment by LSantha** [comment]
@mhaack , in case we need to generate this CSS again from the Venia source code there is a risk that this fix will be lost. This is why I would prefer adding our custom CSS incrementally in a separate file.

**Comment by mhaack** [comment]
@LSantha good point, I moved it to a separate file

File: ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css (hunk 1 above)
Comment (dplaton) [reply ui.apps/src/main/content/jcr_root/apps/venia/components/structure/header/v1/header/clientlibs/css/header-component.css]: A "sticky" element takes space on the page and the rest of the elements are positioned accordingly. A "fixed" element doesn't take space on the page and it remains in that position. 
However, I see Venia uses "sticky" as well so we might as well go with that.

**Pull request merged**